tests/lib/location/src/location_test.c

```diff
@@ -1017,7 +1017,9 @@ void test_location_cellular_cancel_during_ncellmeas(void)
 
 	err = location_request_cancel();
 	TEST_ASSERT_EQUAL(0, err);
-	k_sleep(K_MSEC(1));
+
+	/* Need to wait a bit because no %NCELLMEAS notification is sent after AT%NCELLMEASSTOP. */
+	k_sleep(K_SECONDS(2));
```

Comment:
Maybe try 1200ms (or 1500ms if needed) to optimize the test execution time.

Reply:
Yes, timeout + 100 ms is enough, I'll update this once the timeout has been agreed.